docs/source/_templates/layout.html

```diff
@@ -6,3 +6,69 @@
     </div>
     {% include "searchbox.html" %}
 {% endblock %}
+
+{#
+    ################################################################################
+    # Adding Colab / notebook header like tutorials repo
+    # Based off of
+    # https://github.com/pytorch/pytorch_sphinx_theme/blob/fe1f3d5b9233497d81d04f55f5750ccad92500be/pytorch_sphinx_theme/layout.html#L275-L319
+    ################################################################################
+#}
+
+{%- block content %}
+    {% if pagename.endswith('tutorial') %}
```

Comment:
```suggestion
    {% if 'tutorial' in pagename %}
```

Should do the trick.

<img width="899" alt="Screen Shot 2022-01-11 at 20 11 15" src="https://user-images.githubusercontent.com/855818/149046189-c3d2bf06-7193-4130-aa0d-e9acb11fa275.png">